client/src/components/Creator.js

```diff
@@ -167,7 +184,32 @@ class Creator extends Component {
               </Card.Body>
             </Card>
           </div>
-          <div className="d-flex justify-content-around mt-5 col-md-12">
+          <div class="d-flex justify-content-center mt-5">
```

Comment:
Can we change this to `mt-4` to make the spacing the same between the cards both horizontally and vertically?